docs/specs/openapi.yaml

```
@@ -238,44 +650,361 @@ paths:
+              example:
+                {
+                  "type": "sync",
+                  "status-code": 200,
+                  "status": "OK",
+                  "result": {
+                    "id": "38",
+                    "kind": "autostart",
+                    "summary": "Autostart service \"svc1\" and 1 more",
+                    "status": "Done",
+                    "tasks": [
+                      {
+                        "id": "54",
+                        "kind": "start",
+                        "summary": "Start service \"svc1\"",
+                        "status": "Done",
+                        "progress": {
+                          "label": "",
+                          "done": 1,
+                          "total": 1
+                        },
+                        "spawn-time": "2024-12-27T12:31:26.673287868+08:00",
+                        "ready-time": "2024-12-27T12:31:27.681780702+08:00"
+                      }
+                    ],
+                    "ready": true,
+                    "spawn-time": "2024-12-27T12:31:26.673297951+08:00",
+                    "ready-time": "2024-12-27T12:31:27.686371869+08:00"
+                  }
+                }
+  /v1/notices:
+    get:
+      summary: Get notices
+      tags:
+        - notices
+      description: Get a list of notices that match the filters, ordered by the last-repeated time.
+      parameters:
+        - in: query
+          name: user-id
+          description: Filter notices by user ID (admin only). Only one user ID can be specified.
+          schema:
+            type: integer
+            format: int32
+        - in: query
+          name: users
+          description: If set to "all", return notices for all users (admin only). Cannot be used with `user-id`.
+          schema:
+            type: string
+            enum: ["all"]
+        - in: query
+          name: types
+          description: Filter notices by type. Multiple types can be specified as a comma-separated list.
+          schema:
+            type: array
+            items:
+              type: string
+              enum: [change-update, custom, warning]
+        - in: query
+          name: keys
+          description: Filter notices by keys. Multiple keys can be specified as a comma-separated list.
+          schema:
+            type: array
+            items:
+              type: string
+        - in: query
+          name: after
+          description: Filter notices occurring after the specified timestamp.
+          schema:
+            type: string
+            format: date-time
+        - in: query
+          name: timeout
+          description: The maximum time [duration](#duration) to wait for notices. If no notices are available within this time, an empty list is returned.
+          schema:
+            type: string
+            format: duration
+      responses:
+        "200":
+          description: Successful response.
```

Suggest changing the description for consistency with GET /v1/notices/{id}:
```suggestion
          description: Notices successfully retrieved.
```